netlify/functions/leaderboard.ts

```diff
@@ -39,25 +40,19 @@ const getLeaderboardResults = async (
     .filter((contest) => !contestId || Number(contestId) === contest.contestid);
```

Comment:
GH won't let me select the *actual* line of code since it's not really this PR, but might be nice to give `const filteredContests` a proper type while editing this code — but only if it's a relatively easy lift and doesn't spiral out into a more complicated PR to review!

Reply:
This was an easy lift, there was already a contest type that had a few more things on it then I needed to I just made a simple contest type and extended out to keep the other contest type the same.